src/matlab2tikz.m

```diff
@@ -1911,7 +1915,23 @@ function assertRegularAxes(handle)
     matlabDefaultLineWidth = 0.5;
     if m2t.args.strict ...
             || ~abs(lineWidth-matlabDefaultLineWidth) <= m2t.tol
-        lineOpts = opts_add(lineOpts, 'line width', sprintf('%.1fpt', lineWidth));
```

Comment:
I am unsure, whether this should be part of the strict clause. Normally we make sure this 100% matches matlab so i am unsure how this fits into those custom semantics.